kolibri_explore_plugin/assets/src/views/AssessmentWrapper/ExerciseAttempts/AnswerIcon.vue

```diff
@@ -4,33 +4,34 @@
     <div ref="icon" class="icon">
       <span
         v-if="answer === 'right'"
+        v-b-tooltip:lightbox.hover.right
```

Comment:
This is making a very strong assumption about being inside a container with the ID "lightbox", which seems like it could be trouble down the line. Can we pass this through as a prop (or provide / inject)?

Reply:
Yes actually if we use a bootstrap-vue modal this won't be necessary at all.